src/protocols/rdp/guac_rdpdr/rdpdr_messages.c

```diff
@@ -273,3 +273,15 @@ void guac_rdpdr_process_prn_using_xps(guac_rdpdrPlugin* rdpdr, wStream* input_st
     guac_client_log(rdpdr->client, GUAC_LOG_INFO, "Printer unexpectedly switched to XPS mode");
 }
 
+int guac_rdpdr_encode_utf16(const char* input_string, char* output_string) {
+    int output_length = (strlen(input_string)+ 1) * 2;
+    output_string = realloc(output_string, output_length);
```

Comment:
I'm not sure this is the best way to go about this - feels a little kludgy, but was trying to keep dynamic allocation.

Reply:
Removed.